src/components/timeline/TrackContextMenu.js

```diff
@@ -194,53 +223,157 @@ class TimelineTrackContextMenu extends PureComponent<Props> {
     }
   }
 
-  renderIsolateTrack() {
+  renderIsolateProcess() {
```

Comment:
I split each type of isolation function into its own method, it duplicates a bit of code, but makes it much easier to reason about why to show/hide/enable/disable the item in the menu, which is surprisingly complex.

Reply:
Dismiss the other comment about repeating the code. Yes, you are right :)